exchange/src/exchange/process_spawning/spawning.cpp

```diff
@@ -16,6 +18,16 @@ quote_id(std::string user_id)
 size_t
 spawn_all_clients(nutc::manager::ClientManager& users, SpawnMode mode)
 {
+    const char* wrapper_binary_location = std::getenv("NUTC_WRAPPER_BINARY_PATH");
+    if (wrapper_binary_location == nullptr) {
```

Comment:
```suggestion
    if (!wrapper_binary_location) [[unlikely]] {
```

Reply:
wrapper_binary_location == nullptr is more idiomatic